src/test/java/uk/gov/pay/api/validation/PaymentSearchValidatorTest.java

```diff
@@ -14,46 +14,44 @@
-        PaymentSearchValidator.validateSearchParameters("confirmed", "ref", "2016-01-25T13:23:55Z", "2016-01-25T13:23:55Z");
+        PaymentSearchValidator.validateSearchParameters("confirmed", "ref", "2016-01-25T13:23:55Z", "2016-01-25T13:23:55Z", "1", "1");
     }
 
     @Test
     public void validateParams_reference_shouldGiveAnErrorValidation() throws Exception {
-
         expectedException.expect(aValidationExceptionContaining("P0401", "Invalid parameters: reference. See Public API documentation for the correct data formats"));
 
-        PaymentSearchValidator.validateSearchParameters("confirmed", randomAlphanumeric(500), "2016-01-25T13:23:55Z", "2016-01-25T13:23:55Z");
+        PaymentSearchValidator.validateSearchParameters("confirmed", randomAlphanumeric(500), "2016-01-25T13:23:55Z", "2016-01-25T13:23:55Z", "1", "1");
     }
 
     @Test
     public void validateParams_state_shouldGiveAnErrorValidation() throws Exception {
-
         expectedException.expect(aValidationExceptionContaining("P0401", "Invalid parameters: state. See Public API documentation for the correct data formats"));
 
-        PaymentSearchValidator.validateSearchParameters("invalid", "ref", "2016-01-25T13:23:55Z", "2016-01-25T13:23:55Z");
+        PaymentSearchValidator.validateSearchParameters("invalid", "ref", "2016-01-25T13:23:55Z", "2016-01-25T13:23:55Z", "1", "1");
     }
 
     @Test
     public void validateParams_toDate_shouldGiveAnErrorValidation() throws Exception {
-
         expectedException.expect(aValidationExceptionContaining("P0401", "Invalid parameters: to_date. See Public API documentation for the correct data formats"));
-
-        PaymentSearchValidator.validateSearchParameters("confirmed", "ref", "2016-01-25T13:23:55Z", "2016-01-25T13-23:55Z");
+        PaymentSearchValidator.validateSearchParameters("confirmed", "ref", "2016-01-25T13:23:55Z", "2016-01-25T13-23:55Z", "1", "1");
     }
 
     @Test
     public void validateParams_fromDate_shouldGiveAnErrorValidation() throws Exception {
-
         expectedException.expect(aValidationExceptionContaining("P0401", "Invalid parameters: from_date. See Public API documentation for the correct data formats"));
-
-        PaymentSearchValidator.validateSearchParameters("confirmed", "ref", "2016-01-25T13-23:55Z", "2016-01-25T13:23:55Z");
+        PaymentSearchValidator.validateSearchParameters("confirmed", "ref", "2016-01-25T13-23:55Z", "2016-01-25T13:23:55Z", "1", "1");
     }
 
     @Test
     public void validateParams_shouldGiveAnErrorValidation_forAllParams() throws Exception {
+        expectedException.expect(aValidationExceptionContaining("P0401", "Invalid parameters: state, reference, from_date, to_date, page, display_size. See Public API documentation for the correct data formats"));
+        PaymentSearchValidator.validateSearchParameters("invalid", randomAlphanumeric(500), "2016-01-25T13-23:55Z", "2016-01-25T13-23:55Z", "-1", "-1");
+    }
 
+    @Test
```

Comment:
Is expected a long so I would add a test with a non long value (I would change it to Integer).
- _already changed to Integer, and have a test for non numeric value_ 
Is expected a long from 0? from 1? to n? n has a max value (max Integer value)
_added a test for max value +1 now_
I would check for nonBlank rather than null.
_already done that fix and there is a test for nulls_